cypress/integration/settings-spec.js

```diff
@@ -19,17 +19,15 @@ const aboutWikipediaPage = new AboutWikipediaPage()
 const settingsMenuListEnglishText = [enJson['settings-language'],
   enJson['settings-textsize'],
   enJson['settings-about-wikipedia'],
-  enJson['settings-privacy'],
-  enJson['settings-term'],
-  enJson['settings-about-app']]
+  enJson['settings-about-app'],
+  enJson['settings-privacy-terms']]
 const languageSettingsPopupEnglishText = enJson['language-setting-message']
 const languageChangeDutchText = nlJson['language-change']
 const settingsMenuListDutchText = [nlJson['settings-language'],
   nlJson['settings-textsize'],
   nlJson['settings-about-wikipedia'],
-  nlJson['settings-privacy'],
-  nlJson['settings-term'],
-  nlJson['settings-about-app']]
+  nlJson['settings-about-app'],
+  enJson['settings-privacy-terms']] // TODO: update to nlJson when translations is available
```

Comment:
@jpita the translation for `settings-privacy-terms` is not yet available, I left it here to expect in english because that's what i18n does. Is this fine or is there an actual process to handle something like this?

Reply:
if u want, create the translation in spanish and change the test to change the language to spanish instead of dutch.
that's one way to not leave behind any todos.
or I can do the same for portuguese if you don't have time.

Reply:
Yea that could work, though I thought we are not supposed to change directly any `i18n/*.json` file other than `en.json` and `qqq.json`. But yes will leave this to you once it's in QA if that's cool, that would be helpful and much appreciated

Reply:
> ... I thought we are not supposed to change directly any `i18n/*.json` file other than `en.json` and `qqq.json`.

That's right. If you want to contribute translations to have to create an account on translatewiki.net and do it there.